src/index.ts

```diff
@@ -90,6 +97,73 @@ const isAutoRestMd = (m: md.MarkDownEx) =>
     return t.literal === 'see https://aka.ms/autorest'
   })
 
+const safeLoad = (content: string) => {
+  try {
+    return YAML.safeLoad(content)
+  } catch (err) {
+    return undefined
+  }
+}
+
+export const getDefaultTag = (markDown: commonmark.Node): string => {
+  const startNode = markDown
+  const codeBlockMap = openApiMd.getCodeBlocksAndHeadings(startNode)
+
+  const latestHeader = 'Basic Information'
+  const lh = codeBlockMap[latestHeader]
+  if (lh && lh.literal) {
+    const latestDefinition = safeLoad(lh.literal)
+    if (latestDefinition && latestDefinition.tag) {
+      return latestDefinition.tag
+    }
+  }
+  for (const idx of Object.keys(codeBlockMap)) {
+    const block = codeBlockMap[idx]
+    if (!block || !block.info || !block.literal || !/^(yaml|json)$/.test(block.info.trim().toLowerCase())) {
+      continue
+    }
+    const latestDefinition = safeLoad(block.literal)
+    if (latestDefinition && latestDefinition.tag) {
+      return latestDefinition.tag
+    }
+  }
+  return ''
```

Comment:
Why return an empty string? If not found, return undefined

Reply:
This case, it's fine to use empty string to indicate not found